### ISSUE TITLE
Effective Java - favor static classes over nonstatic

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#1501]
All non-static nested classes are rewritten to static. In fact, this is almost complete so far, and only two classes haven't been rewritten.

#### Class/Reason

 - [x] CoverArtController
   -  The start had been postponed because there was no test at all ... It's okay now!!
 - [x] PlayQueueInputStream
   -  After a while, I was thinking about the possibility of removing the inner class. (The same operation is possible without it)
       - The performance of the start playing process performed by PlayQueueInputStream$Prepare is relatively heavy. Especially for high resolution transcoding.
         - If verbose logging is enabled, logging the thread name of the pooling that is running (for convenience in memory profiling. If you do not execute it in the pooling thread, it will be one of the http threads, so it is difficult to understand). Debugging assistant.
         - Jpsonic's PlayQueueInputStream$Prepare has MIN thread priority. For Airsonic and Subsonic, transcoding priority is NORMAL. Even if transcoding or HTTP access(NORMAL) is overcrowded
   -  Simply put, it was a bit experimental code. This seems to be fine as it is. The design has a few advantages, except that the source code is cumbersome.
